core/urls.py

```diff
@@ -13,9 +13,10 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
+from django.conf.urls import url, include
 from django.contrib import admin
 
 urlpatterns = [
+    url(r'^grappelli/', include('grappelli.urls')),  # grappelli URLS
```

Comment:
@andela-aoduola: Though used in the setup guideline, I think the comment is really not necessary. Also, I think the default module docstring should be cleared.